### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by isht3**
Landing page for evalai-cli.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+            </ol>
+            <br>
+            <div class="row">
+                <div class="col-md-12" id="code-label">
+                    <h4 class="code-container-style"><b>Configure the Host URL (OPTIONAL)</b></h4>
+                    <p>Only run the following command if you want to use CLI with your forked version of EvalAI.</p>
+                </div>
+            </div>
+            <div class="command">
+                <div class="label">Run this command:</div>
```
Comment (RishabhJain2018) [inline docs/index.html]: please don't make it bold.

**Comment by RishabhJain2018** [comment]
@isht3 Any update on this one?

**Comment by isht3** [comment]
@RishabhJain2018 Changes made.

**Comment by isht3** [comment]
@RishabhJain2018 Can we merge this?

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+        <ul class="primary-nav">
+            <li><a class="github" href="https://github.com/Cloud-CV/EvalAI-CLI">GitHub</a></li>
+        </ul>
+    </footer>
+    <script src="./assets/jquery.min.js"></script>
+    <script src="./assets/bootstrap.min.js"></script>
+    <script src="./assets/active.js"></script>
+</body>
+
+</html>
```
Comment (RishabhJain2018) [inline docs/index.html]: Please add a new line here!

File: docs/assets/github-markdown.css [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
...V8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
```
Comment (RishabhJain2018) [inline docs/assets/github-markdown.css]: Why is this long string?

File: docs/assets/github-markdown.css [hunk 4]
```diff
@@ -0,0 +1,708 @@
...
+}
+
+.markdown-body .task-list-item input {
+  margin: 0 0.2em 0.25em -1.6em;
+  vertical-align: middle;
+}
+
+.markdown-body hr {
+  border-bottom-color: #eee;
+}
```
Comment (RishabhJain2018) [inline docs/assets/github-markdown.css]: Please add a new line!

**Comment by isht3** [comment]
@RishabhJain2018 Changes made!

**Comment by RishabhJain2018** [comment]
@isht3 Can you please also paste the complete screenshot of the docs page?

**Pull request merged**